extensions/wikia/PageHeader/styles/index.scss

```diff
@@ -23,5 +23,12 @@
 		border: 0;
 		height: 1px;
 	}
-}
 
+	&__counter {
+		color: $color-text;
+		float: right;
```

Comment:
How about tackling it during the work on buttons? Right now it looks good, when we add the buttons, it will probably stop looking good and then the flex will be a must.